packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/nodes/NodeSizeDetailsTests.java

```diff
@@ -20,17 +26,147 @@ public class NodeSizeDetailsTests extends RestTest
     private UserModel user1;
     private SiteModel siteModel;
     private FolderModel folder;
+    private String jobId;
 
     @BeforeClass(alwaysRun = true)
     public void dataPreparation()
     {
-        user1 = dataUser.createRandomTestUser("User-1");
+        user1 = dataUser.getAdminUser();
```

Comment:
Does it mean that only `admin` users can calculate the folder size?

Reply:
No, the Random Test User is also able to calculate the folder size. 
We have updated it to use RandomUser instead.